src/mach/exports.rs

```diff
@@ -294,6 +294,30 @@ impl<'a> ExportTrie<'a> {
             location,
         }
     }
+
+    /// Create a new, lazy, zero-copy export trie from the `LinkeditDataCommand` `command`
+    pub fn new_from_linkedit_data_command(
+        bytes: &'a [u8],
+        command: &load_command::LinkeditDataCommand,
```

Comment:
Unrelated to your PR< but i just noticed the capitalization for this is strange, surprised it's not LinkEditDataCommand; is this how it occurs in mach sources?

Reply:
Given it's `LINKEDIT`, not `LINK_EDIT`, I think the current capitalization makes sense?